### ISSUE TITLE
Add link to project page in simulator

### ISSUE THREAD

**Issue by satu0king** [CircuitVerse/CircuitVerse#1729]
Add a link to redirect to project page directly from the simulator.
Please send PR to new_UI branch. 

![image](https://user-images.githubusercontent.com/2092958/94590655-a5885700-02a4-11eb-9db8-1973ec46eb43.png)

**Comment by GAUTAMSAHARAN** [comment]
can i work on this feature

**Comment by satu0king** [comment]
> can i work on this feature

Go ahead!

**Comment by GAUTAMSAHARAN** [comment]
should I directly put href in <a> tag or we need to go with creating function in js just like with other options in dropdown.

**Comment by GAUTAMSAHARAN** [comment]
One more thing, rather than adding another option in menu how about we put link directly on project name in the navbar

**Comment by GAUTAMSAHARAN** [comment]
I have created this feature i put link on the project-name in simulator page, and made a pr . Please review it.

**Comment by satu0king** [comment]
> One more thing, rather than adding another option in menu how about we put link directly on project name in the navbar

Interesting, However, I think it is better if we add the hyperlink on a icon like this.
![image](https://user-images.githubusercontent.com/2092958/94779298-577a6d00-03e4-11eb-8890-219b85697f7a.png)

@Shivansh2407, @sal2701 any inputs?

**Comment by GAUTAMSAHARAN** [comment]
What i think is we don't have any kind of icon in whole nav-bar so it looks bit different, and usually we reach to our desired location by clicking on main title, that's my thought.

**Comment by satu0king** [comment]
> What i think is we don't have any kind of icon in whole nav-bar so it looks bit different, and usually we reach to our desired location by clicking on main title, that's my thought.

Agreed with your point on the icon. However, there is another feature coming in soon letting users edit the the project name by clicking on the project name (Like in google docs). This feature takes precedence over yours. Moreover, the userwill not expect the project name to be a hyperlink. I think for now just add the hyperlink to the drop down. We can always move it as we improve. Thanks!

**Comment by GAUTAMSAHARAN** [comment]
ok, I understand and I put a hyperlink in dropdown menu rather than in project-title. Now you can review my PR. Thanks for your help.
